vidarr-niassa/src/main/java/ca/on/oicr/gsi/vidarr/niassa/NiassaWorkflowEngine.java

```diff
@@ -304,6 +311,37 @@ public JsonNode run(
     return null;
   }
 
+  @Override
+  public void startup() {
+    // Set up Postgres config if it has not been configured already
```

Comment:
This says 'if it has not been configured already' but the check for pgDataSource being null is gone

Reply:
Fixed the comment. `startup` will only get run once.